### PR TITLE
Update to LangChain4j 0.34

### PR DESCRIPTION
**Pull request by geoand**
- Closes: #873

### CODE REVIEW

File: docs/modules/ROOT/pages/includes/quarkus-langchain4j-infinispan.adoc [hunk 1]
```diff
@@ -10,23 +10,6 @@ h|[[quarkus-langchain4j-infinispan_configuration]]link:#quarkus-langchain4j-infi
 h|Type
 h|Default
 
-a|icon:lock[title=Fixed at build time] [[quarkus-langchain4j-infinispan_quarkus-langchain4j-infinispan-client-name]]`link:#quarkus-langchain4j-infinispan_quarkus-langchain4j-infinispan-client-name[quarkus.langchain4j.infinispan.client-name]`
```
Comment (jmartisk) [inline docs/modules/ROOT/pages/includes/quarkus-langchain4j-infinispan.adoc:13]: How come this is disappearing from the docs when we didn't remove the related property?

Comment (jmartisk) [reply docs/modules/ROOT/pages/includes/quarkus-langchain4j-infinispan.adoc:13]: And the same question for huggingface and BAM docs

Comment (geoand) [reply docs/modules/ROOT/pages/includes/quarkus-langchain4j-infinispan.adoc:13]: I have no idea whatsover

Comment (geoand) [reply docs/modules/ROOT/pages/includes/quarkus-langchain4j-infinispan.adoc]: I rebased and it looks better now

**Comment by geoand** [comment]
All 🟢

**Review by jmartisk — APPROVED** [review]
Awesome!

**Pull request merged**